### DISCUSSION TITLE
Add DVD, DVD menu support with libdvdcss, libdvdnav, libdvdread

### DISCUSSION THREAD

**Discussion by ghost** [linuxdeepin/developer-center#2946] [Features Request & Ideas | 特性请求 & 头脑风暴]
There is no support (playback and DVD menu) in Deepin Movie for encrypted DVD and DVD folders.

These things are missing libdvdcss, libdvdnav, libdvdread.

Can you add the support?

There is no wonderful movie player on Linux that has these features.

**Comment by CodingEagle2025** [comment]
I understand that libdvdcss has copyright issues，https://tech.qq.com/a/20130303/000043.htm。

**Comment by ghost** [comment]
@jhkyy 
libdvdcss is just a library and is licensed under GPL v2. You can integrate it, but by default you can leave it disabled and if users want to enable it they have to manually enable it in Deepin Movie settings (libdvdcss (on/off).

It should work out of the box.

**Comment by meiyixiang** [comment]
Problems have been fed back to the product and have been adopted to schedule implementation
